TombEngine/Game/items.h

```diff
@@ -138,20 +141,23 @@ struct ItemInfo
 	BitField MeshBits	  = BitField();
 
 	unsigned short Flags; // ItemFlags enum
-	short ItemFlags[8];
+	short ItemFlags[NUM_ITEM_FLAGS];
 	short TriggerFlags;
 
 	// TODO: Move to CreatureInfo?
 	uint8_t AIBits; // AIObjectType enum.
 	short AfterDeath;
 	short CarriedItem;
 
-	bool TestOcb(short ocbFlags);
+	bool TestOcb(short ocbFlags) const;
 	void RemoveOcb(short ocbFlags);
 	void ClearAllOcb();
-
-	bool TestFlags(short id, short value);
-	void SetFlags(short id, short value);
+	
+	bool  TestFlags(int id, short flags) const;		// ItemFlags[id] & flags
```

Comment:
What's the difference between `TestFlags` and `TestFlagField`? This is getting confusing.

Reply:
TestFlags checks individual flags (& operation). TestFlagField checks whether the packed bit fields are the same (== operation). Old names were even more confusing. What might be a better alternative?

Reply:
Honestly, I don't know. Introducing a set of functions to work with already exposed `ItemFlags` looks weird to me, but it's already done, so maybe it should be kept as it is.